include/oneapi/dpl/pstl/hetero/dpcpp/utils_ranges_sycl.h

```diff
@@ -341,7 +341,7 @@ struct __get_sycl_range
 
     template <typename _Iter>
     buf_type<val_t<_Iter>>
-    copy_back(_Iter __first, buf_type<val_t<_Iter>> buf, /*_copy_back*/ ::std::false_type)
+    copy_back(_Iter, buf_type<val_t<_Iter>> buf, /*_copy_back*/ ::std::false_type)
```

Comment:
on the implementation level it look very weird when we have different number of parameters (conceptually) but still introduce tags to those overloads